anti.md

```diff
@@ -58,12 +127,53 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think this is where this idea really falls flat on its face. I think that
+most of the work of this project would be on dealing with the course
+registration system. The language would probably some sort of cross between a
+CLI and SQL, and even if it were done graphically it would probably have a
+similar core.
+
+The result is that I think most of the time on this project would be spent
+trying to successfully interact with Portal, which is definitely on the
+**systems** side of things.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think this would actually be a pretty monumental project from the systems
+side. Interacting with portal is notoriously impossible to do. This project
+might actually end up being forbidden by CIS or the Registrar's Office because
+it could potentially mess up students' registration if it had a bug in a
+critical situation.
+
+From a language perspective it may actually be a pretty unambitious project. It
+seems like some SQL type language would be appropriate for searching, and the
+other features would be equally simple.
+
+One thing that might be interesting is setting up a GUI for the SQL queries. I
+could imagine typing and having boxes appear, color coordinated by constraint
+type.
+
+One could make the project more linguistically ambitious by adding a rules
+feature (register for class X if it opens), and then trying to figure out the
+best way for users to interact with this feature.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+I think that the most appealing part of the project would be the potential
+impact. We see a lot of students around us struggle with Portal every time
+class registration rolls around, and it would be nice to be able to fix that.
+
+I also find that building CLIs is a very satisfying thing, and ends up having a
+larger impact than you might think because then people can use Unix command
+line tools to expand your tool in ways that you did not expect. In some sense
+you end up making an API that others can use in bigger and better ways.
+
+However, I think this project has a pretty substantial drawback, which is that
+it requires only a small amount of linguistic work, and a very large amount of
+work behind the scenes to make it function. It's also questionable whether
+changing the interface to an existing service is really making a language. I
+would say yes because you are changing the way users deliver instructions to
+computers, but it could be debated.
```

Comment:
Overall, I don't know that I'd call this an example of where a DSL would be the wrong tool for the job.  I think you articulate the drawbacks well though.